lib/fastlane_core/project.rb

```diff
@@ -114,7 +122,10 @@ def select_scheme
       if schemes.count == 1
         options[:scheme] = schemes.last
       elsif schemes.count > 1
-        if Helper.is_ci?
+        if ENV["AUTOMATED_SCHEME_SELECTION"] and schemes.include? project_name
+          Helper.log.info ("Using the scheme matching the project name (" + project_name + ").").yellow
```

Comment:
Also, if you use variable interpolation inside the string, like "... name #{project_name}." then the parentheses won't be needed to turn the whole message yellow :+1: